message.go

```diff
@@ -82,6 +88,7 @@ func (c *Client) MessageWrite(ctx context.Context, r MessageWriteRequest) (*Mess
 	return &res, nil
 }
 
+// Messages performs a Get request.
 func (c *Client) Messages(ctx context.Context, r MessagesRequest) (*MessagesReply, error) {
```

Comment:
What does this get request do, return a a list of messages? Could do with a bit more detail for folk like me who don't know what's going on haha